lib/AST/Expr.cpp

```diff
@@ -1974,10 +1974,10 @@ FORWARD_SOURCE_LOCS_TO(ClosureExpr, Body.getPointer())
 
 Expr *ClosureExpr::getSingleExpressionBody() const {
   assert(hasSingleExpressionBody() && "Not a single-expression body");
-  auto body = getBody()->getFirstElement();
+  auto body = getBody()->getElements().back();
```

Comment:
```suggestion
  auto body = getBody()->getLastElement();
```

Ditto for other usages.

Reply:
I converted all usages!